### ISSUE TITLE
[loss = loss_xy + loss_wh + loss_obj + loss_cls] ...  Did you lost the loss_l2?

### ISSUE THREAD

**Issue by whuzt** [Tianxiaomo/pytorch-YOLOv4#90]

**Comment by oreo-lp** [comment]
lost_l2 is just the error between predictions and ground truth. it does't make a contribution to loss function.

**Comment by whuzt** [comment]
OK， thank you for your reply
